packages/core/src/app/Exporter.ts

```diff
@@ -86,6 +86,8 @@ export interface Exporter {
     signal: AbortSignal,
   ): Promise<void>;
 
+  mergeMedia?(): Promise<void>;
```

Comment:
I don't think it's immediately clear what this is for. Maybe we could add a comment.